mifosng-android/src/main/java/com/mifos/mifosxdroid/activity/PathTrackingActivity.java

```diff
@@ -90,4 +118,51 @@ public void onServiceDisconnected(ComponentName componentName) {
         start.setEnabled(false);
         stop.setEnabled(false);
     }
+
+    private boolean buildLocationEnableDialog() {
+
+        LocationManager locationManager =
+                (LocationManager) getApplicationContext()
+                .getSystemService(Context.LOCATION_SERVICE);
+        boolean gpsEnabled = false;
+        boolean networkEnabled = false;
+
+        try {
+            gpsEnabled = locationManager.isProviderEnabled(LocationManager.GPS_PROVIDER);
+        } catch (Exception ex) {
+        }
+
+        try {
+            networkEnabled = locationManager.isProviderEnabled(LocationManager.NETWORK_PROVIDER);
+        } catch (Exception ex) {
+        }
+
+        if (!gpsEnabled && !networkEnabled) {
+            // notify user
+            final AlertDialog.Builder builder = new AlertDialog.Builder(this);
```

Comment:
Please use https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MaterialDialog.java for showing material dialog.